src/components/extracrewdetails.tsx

```diff
@@ -129,6 +226,48 @@ class ExtraCrewDetails extends Component<ExtraCrewDetailsProps, ExtraCrewDetails
 		);
 	}
 
+	renderPolestars() {
+		if (!this.state.polestars) {
```

Comment:
If the crew doesn't have a keystone, they are likely not in Portal and therefore not retrievable (excepting 1/2/3 rarity crew, who I don't think people care about retrieving for the most part).

```suggestion
		if (!this.state.polestars || !this.state.keystone) {
```